xml/pacemaker-schemas.pc.in

```diff
@@ -0,0 +1,12 @@
+# compatible with pacemaker-cib.pc:schemadir
+schemadir=@CRM_SCHEMA_DIRECTORY@
+
+# where machine-friendly API interchange format is formalized as schemas
+schemadir4api=${schemadir}/api
+
+Name:             Pacemaker Schemas & Transformations
```

Comment:
Following common practice, the name should be something simple like pacemaker-schemas

Reply:
It's anticipating the (very real) future amendments, so that the label
is not immediately tilted again.

Reply:
Plus that's not something lexically restricted, has merely informative
character to the user.

See also:
dotnet35.pc: Standard libraries in a .NET setup